HWs 4, 5/yurii_1995/additional_task2.py

```diff
@@ -0,0 +1,13 @@
+# class work 3, task 2
+
+try:
+
+    number = int(input("Type a number "))
+    
+    if number % 2 == 0:
+        print("The given number is even")
+    elif number % 2 == 1:
+        print("The given number is odd")
```

Comment:
if number % 2 == 0:
        print("The given number is even")
    else:
        print("The given number is odd")